docs/Migrating.md

```diff
@@ -24,6 +24,12 @@ The versions can be found in
 * `BrainParameters` and `SpaceType` have been removed from the public API
 * `BehaviorParameters` have been removed from the public API.
 * `DecisionRequester` has been made internal (you can still use the DecisionRequesterComponent from the inspector). `RepeatAction` was renamed `TakeActionsBetweenDecisions` for clarity.
+* The following methods in the `Agent` class have been deprecated. They will be removed in a later release:
```

Comment:
Can we change this to something like "The following methods in the `Agent` class have been renamed. The original method names will be removed in a later release."

The current sentence sounds like all the methods will be deprecated (including the renamed one)

Reply:
```suggestion
* The following methods in the `Agent` class have been renamed. The original method names will be removed in a later release:
```